js/samples/simple/Readme.md

```diff
@@ -0,0 +1,20 @@
+# Immersive Reader - Simple HTML/Javascript Sample
```

Comment:
This sample is just showing how to use HTML, right? Don't think we need to include the options in this one

Reply:
correct, options usage removed